src/app/match-viewer-app/summoner-tile/summoner-tile.component.html

```diff
@@ -1,44 +1,108 @@
-<div
-  class="summoner-tile"
-  fxLayout="row"
-  fxLayoutGap="20px"
-  fxLayoutAlign="start center">
-  <img
-    class="summoner-champion-icon"
-    [src]="summonerInfo.champion.championIcon" />
-  <div
-    class="summoner-spells"
-    fxLayout="column"
-    fxLayoutGap="5px">
-    <img
-      class="summoner-spell"
-      [src]="summonerInfo.firstSpellIcon.url"
-      [alt]="summonerInfo.firstSpellIcon.alt">
-      <img
-      class="summoner-spell"
-      [src]="summonerInfo.secondSpellIcon.url"
-      [alt]="summonerInfo.secondSpellIcon.alt">
-  </div>
-  <div
-    class="summoner-info"
-    fxLayout="column">
-    <div class="summoner-name">
-      {{summonerInfo.summonerName}}
-    </div>
-    <div
-      class="summoner-detail"
-      fxLayout="row" fxLayoutGap="20px">
-      <div class="summoner-kda-view" fxLayout="row">
-        <div class="kda-label">KDA</div>
-        <div class="kda">
-          {{summonerInfo.summonerKDA.kill}} <b>/</b>
-          {{summonerInfo.summonerKDA.death}} <b>/</b>
-          {{summonerInfo.summonerKDA.assist}}
+<!-- Blue Team Tile -->
```

Comment:
Blue team & red team separate tile seems like redundant code, is this the best way?

Reply:
Added conditional classes that will render the summoner-tile accordingly, however, readability of code is not as clear as before due to possible confusion in logic of code.

Reply:
Oh... dang oh well. We could have discussed it if you thought it was more important, it looked a little bloated but since it was just one small component I guess it might have been a better option. This is good though! Thanks